<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeechToTextDialog</class>
 <widget class="QDialog" name="SpeechToTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>236</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>236</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>236</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>415</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SpeechToText</string>
  </property>
  <widget class="QPushButton" name="StartPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PausePushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QComboBox" name="langCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>310</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="SpeechTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="StopPushButton">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language : </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ExportPathTextEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>270</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowserPushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ExportCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Export To </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
